### DISCUSSION TITLE
Set offline agent to uninstall on next connection

### DISCUSSION THREAD

**Discussion by Tomatoschewps** [Ylianst/MeshCentral#6581] [Q&A] [answered]
Hi,
Is there any solution do indicate Meshcentral to uninstall an agent next time he is seen by server ?
We have some agents on PC who doesn't connect during the day or once every X week so until now we haven't got the chance to uninstall them and even if we delete them on Meshcentral, they got created again next time they got online.
Having a special group for agents to be uninstalled would be much appreciated.

**Comment by si458** [comment, marked as answer]
Unfortunately not this would need to be like a feature request to add this feature in
The only way at the moment is when the device is online, go into the web ui, then the device, actions, then uninstall

**Reply by Tomatoschewps** [reply]
Ok, thanks.
I'll post a feature request for this.
